scripts/provisioning/setupcmd.py

```diff
@@ -594,24 +594,22 @@ def get_config_param_for_BG_delete_account(self):
 
     return bgdelete_acc_input_params_dict
 
-  def delete_replication_config(self):
-    conn = ldap.initialize("ldapi://")
-    conn.sasl_non_interactive_bind_s('EXTERNAL')
-
-    dn = "cn=config"
-    self.deleteattribute(conn, dn, "olcServerID")
-
-    dn = "olcDatabase={0}config,cn=config"
-    self.deleteattribute(conn, dn, "olcSyncrepl")
-    self.deleteattribute(conn, dn, "olcMirrorMode")
-
-    dn = "olcDatabase={2}mdb,cn=config"
-    self.deleteattribute(conn, dn, "olcSyncrepl")
-    self.deleteattribute(conn, dn, "olcMirrorMode")
-
   def deleteattribute(self, conn, dn, attr_to_delete):
     mod_attrs = [(ldap.MOD_DELETE, attr_to_delete, None)]
     try:
       conn.modify_s(dn, mod_attrs)
     except:
       self.logger.info('Attribute '+ attr_to_delete + ' is not configured for dn '+ dn)
+
+  def modify_attribute(self, dn, attribute, value):
```

Comment:
why can't we directly import from py-utils ?
we can avoid re-writing the same function

Reply:
plz suggest how to do it.

Reply:
Fixed

Reply:
Tried but its failing with nested used packages dependency